src/helper_functions.hpp

```diff
@@ -44,3 +44,12 @@
 
  */
 std::string string_from_unterminated(const char* data, size_t data_length);
+
+/// Convert a wide string to a UTF8 encoded string
+///
+/// @param[in] wStr  Unicode encoded wide string
+///
+/// @return UTF-8 encoded string
+///
+/// @throw std::range_error when `wStr` contains an invalid code point
```

Comment:
Is this comment up-to-date? I did not see any test validating that statement. By reading the tests it seems we are still returning something when `wStr` contains a invalid code point.

Reply:
Nope, that comment is wrong.